Sources/DCoreKit/Model/Operations/NftCreateOperation.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import BigInt
+
+public struct NftCreateOperation: Operation {
+
+    public var symbol: String {
+        willSet { precondition(Asset.hasValid(symbol: symbol), "invalid nft symbol: \(symbol)") }
+    }
+    public var options: NftOptions {
+        willSet {
+            precondition(
+                options.description.count <= DCore.Constant.uiaDescriptionMaxChars,
+                "description cannot be longer than \(DCore.Constant.uiaDescriptionMaxChars) chars"
+            )
+            precondition(options.maxSupply <= 0xFFFFFFFF)
+        }
+    }
+    public var definitions: [NftDataType] {
+        willSet {
+            precondition(
+                definitions.allSatisfy { $0.modifiable == NftDataType.ModifiableBy.nobody || !$0.name.isNil() },
```

Comment:
$0.modifiable == NftDataType.ModifiableBy.nobody || !$0.name.isNil()  is this condition correct?